examples/simple_repeater/MyMesh.cpp

```diff
@@ -173,6 +173,12 @@ int MyMesh::handleRequest(ClientInfo *sender, uint32_t sender_timestamp, uint8_t
 
     telemetry.reset();
     telemetry.addVoltage(TELEM_CHANNEL_SELF, (float)board.getBattMilliVolts() / 1000.0f);
+
+    float temperature = (float)board.getMCUTemperature();
+    if(!isnan(temperature)) { // Supported boards with built-in temperature sensor. ESP32-C3 may return NAN
+      telemetry.addTemperature(TELEM_CHANNEL_SELF, (float)board.getMCUTemperature()); // Built-in MCU Temperature
```

Comment:
Looks like you're calling `board.getMCUTemperature()` twice, instead of using the `temperature` variable that already did a read? Since those methods also return `float` type, probably don't need the type casts?